packages/agoric-cli/src/bin-agops.js

```diff
@@ -74,13 +73,11 @@ program.addCommand(makeAuctionCommand(logger, { ...procIO, fetch }));
 program.addCommand(makeInterCommand(procIO, { fetch }));
 program.addCommand(makeTestCommand(procIO, { fetch }));
 
-try {
-  await program.parseAsync(process.argv);
-} catch (err) {
+program.parseAsync(process.argv).catch(err => {
```

Comment:
```suggestion
E.when(
  program.parseAsync(process.argv),
  undefined,
  err => {
```
Makes me wonder if we should have an `E.catch` :/